community/index.md

```diff
@@ -1,158 +1,4 @@
 ---
-title: Apache Druid (incubating) Community
-sectionid: community
-layout: simple_page
+layout: redirect_page
+redirect_target: 'https://druid.apache.org/community'
 ---
-
-## Community
```

Comment:
A Markdown doc is leaving here and generated HTML is entering the ASF website repo. I think that's fine for this test, but before migrating more pages we should figure out how to get a Markdown build working in the ASF repo.

Reply:
Agreed